perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -83,6 +84,20 @@
                      'Allow swiftclient to access Swift service without \n'
                      'having to verify the SSL certificate')
 
+flags.DEFINE_integer('objects_per_stream', 1000,
```

Comment:
Yes, and I'll also prefix them with 'object_storage', which I forgot to do before!